### ISSUE TITLE
[PATTERN STRUCTURE] add version field in pattern instance json file

### ISSUE THREAD

**Issue by compaluca** [testable-eu/sast-testability-patterns#29]
## Problem statement
Patterns have a different level of maturity. At the moment we are keeping track of this via some manual activity (e.g., table in an md file) and little scripts creating the pattern tables in the main [readme file](https://github.com/testable-eu/sast-testability-patterns).

Everytime an important change is done in a pattern, those pattern table would only be changed if the table in the specific pattern review issue is also modified.

This is thus difficult to maintain. 

## Proposed changes

However, when we do a pattern review we act on the json files of that pattern anyhow, so we could control there the version by adding a `version` field. 

@SoheilKhodayari: what do you think?

**Comment by SoheilKhodayari** [comment]
Right, that's a good idea. We can add a `version` field to each pattern json file and create a script to re-generate the table in the readme.
